homeassistant/components/qwikswitch/__init__.py

```diff
@@ -225,7 +227,7 @@ def callback_qs_listen(qspacket):
                 async_dispatcher_send(hass, qspacket[QS_ID], qspacket)
 
         # Update all ha_objects
-        hass.async_add_job(qsusb.update_from_devices)
+        hass.async_add_hass_job(update_job)
```

Comment:
I feel like outside of helpers, we should not even use the `add_hass_job` functions, but instead just call directly executor etc.

Reply:
Good call. It makes more sense to use `async_create_task` here